### PR TITLE
Status field created on new forms

### PR DESCRIPTION
**Pull request by Jdallos**

### CODE REVIEW

File: src/utils/metaform-utils.ts [hunk 1]
```diff
@@ -435,6 +435,60 @@ namespace MetaformUtils {
...
+
+  /**
+   * Filter out status from form
+   * 
+   * @param form Metaform form
+   * @returns Metaform without status field
+   */
+  export const removeStatusFieldFromForm = (form: Metaform) => {
+    const sectionsWithoutStatusField = form.sections?.map(section => {
+      const hasStatusField = section?.fields?.filter(field => field.name !== "status");
```
Comment (EVP1989) [inline src/utils/metaform-utils.ts:471]: Naming seems misleading, should it be like hasNoStatusField?

**Review by EVP1989 — COMMENTED** [review]
Added comment

File: src/utils/metaform-utils.ts (hunk 1 above)
Comment (Jdallos) [reply src/utils/metaform-utils.ts:471]: yep

**Review by EVP1989 — APPROVED** [review]
Approved

**Pull request merged**